components/support/ffi/src/handle_map.rs

```diff
@@ -685,6 +691,7 @@ unsafe impl IntoFfi for Handle {
 /// // pub extern "C" fn mylib_destroy_thing(h: u64, err: &mut ExternError)
 /// define_handle_map_deleter!(ITEMS, mylib_destroy_thing);
 /// ```
+#[derive(Default)]
 pub struct ConcurrentHandleMap<T> {
```

Comment:
Can we implement `new` in terms of `ConcurrentHandleMap::default()`?

Reply:
No, apparently the auto-derived Default has a T: Default bound. I've changed it to implement Default using new() though.